cnf-tests/Dockerfile

```diff
@@ -22,6 +22,20 @@ WORKDIR $TESTER_PATH
 
 RUN go build -mod=vendor -o /sctptest
 
+# build hugepages-allocator's binary
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.19 AS builder-hugepages-allocator
+
+ENV PKG_NAME=github.com/openshift-kni/cnf-features-deploy
+ENV PKG_PATH=/go/src/$PKG_NAME
+ENV TOOL_PATH=$PKG_PATH/cnf-tests/pod-utils/hugepages-allocator
+
+RUN mkdir -p $PKG_PATH
+
+COPY . $PKG_PATH/
+WORKDIR $TOOL_PATH
+
+RUN go build -mod=vendor -o /hugepages-allocator
+
 # build latency-test's runner binaries
 FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.22-openshift-4.18 AS builder-latency-test-runners
```

Comment:
Nope. my bad

Reply:
Actually yes, this is the standard `Dockerfile` not the `Dockerfile.openshift` I wasn't sure whether we want to change images here as well, because the cnf-tests image is centos7 based

Reply:
I must say I am not sure who use this file but if you can please update everything to rhel 9 base.

I don't think this dockerfile really work as we build stuff on rhel 8 and 9 but put everything inside a centos7 image...